src/Illuminate/Support/Optional.php

```diff
@@ -59,6 +59,25 @@ public function __call($method, $parameters)
         }
     }
 
+    /**
+     * Dynamically check a property exists on the underlying object.
+     *
+     * @param $name
```

Comment:
invalid syntax

Reply:
This is auto generated by PhpStorm, I didn't notice that. Since it has been merged, should I open another PR?